client/wordpress-com.js

```diff
@@ -108,9 +108,9 @@ const sections = [
 		group: 'sites'
 	},
 	{
-		name: 'settings-seo',
-		paths: [ '/settings/seo' ],
-		module: 'my-sites/site-settings/seo-settings',
+		name: 'settings-traffic',
+		paths: [ '/settings/traffic' ],
+		module: 'my-sites/site-settings/traffic',
```

Comment:
Good call. 

I was actually planning to move the SEO settings bundle to the settings bundle, but seeing the current SEO settings bundle size quickly convinced me not to do so.

So, what we're doing here is essentially:

* Renaming the SEO Settings bundle to Traffic.
* Including Analytics within the Traffic bundle.

Here is a comparison of the bundle sizes in master and this branch:

**this branch**
`settings`: 951 kB
`settings-traffic`: 580 kB

**master**
`settings`: 963 kB
`settings-seo`: 531 kB

In the future we'll probably have to include some other stuff in that bundle, so it'll probably grow a bit more. See #11615 for more details on that.

One of the next things I'm planning is to attempt splitting the settings bundle in smaller bundles, as discussed in p4TIVU-5Zs-p2, so more work in that direction is coming soon.